src/ToricVarieties/LineBundles/constructors.jl

```diff
@@ -48,7 +48,37 @@ end
 
 
 ########################
-# 3: Display
+# 3: Special constructors
+########################
+
+# The following is both a special constructor of a toric line bundle and an attribute of a toric variety.
+# I consider as the latter, hence the function name is not capitalized. Also, I cache the result.
```

Comment:
General remark: our users won't be Julia experts in general (hopefully -- if only Julia experts end up using Oscar, we did something wrong). Therefore, we can't expect them to understand the subtle differences between a constructor and a "regular" function that just happens to return a "new" object.

Thus we decided long ago that ideally all constructors should also be accessible under a suitable `snake_case` name. Indeed, even for Julia experts it can be difficult to guess whether to use, say, `StructureSheaf` or `structure_sheaf` in case both exist.